Vienna/Sources/Main window/BrowserTab.swift

```diff
@@ -230,17 +230,11 @@ extension BrowserTab: Tab {
 
     func back() -> Bool {
         let couldGoBack = self.webView.goBack() != nil
-        // title and url observation not triggered by goBack() -> manual setting
-        self.url = self.webView.url
-        updateTabTitle()
```

Comment:
Seems like that code has been necessary at some point, maybe an old macOS version? If it is not necessary in the current version we should maybe just guard it with the OS version.

Reply:
I have been able with the oldest macOS version we currently support (High Sierra (10.13.6)) and it behaves correctly without this code.